tests/api/plugins/content-releases/releases.test.api.ts

```diff
@@ -0,0 +1,646 @@
+'use strict';
+
+import { createStrapiInstance } from 'api-tests/strapi';
+import { createAuthRequest } from 'api-tests/request';
+import { describeOnCondition } from 'api-tests/utils';
+import { createTestBuilder } from 'api-tests/builder';
+
+const edition = process.env.STRAPI_DISABLE_EE === 'true' ? 'CE' : 'EE';
+
+const productUID = 'api::product.product';
+const productModel = {
+  draftAndPublish: true,
+  pluginOptions: {},
+  singularName: 'product',
+  pluralName: 'products',
+  displayName: 'Product',
+  kind: 'collectionType',
+  attributes: {
+    name: {
+      type: 'string',
+    },
+    description: {
+      type: 'string',
+      required: true,
+    },
+  },
+};
+
+describeOnCondition(edition === 'EE')('Content Releases API', () => {
+  const builder = createTestBuilder();
+  let strapi;
+  let rq;
+  const data = {
+    releases: [],
+  };
+
+  const deleteAllReleases = async () => {
+    const releases = await rq({
+      method: 'GET',
+      url: '/content-releases',
+    });
+
+    await Promise.all(
+      releases.body.data.map(async (release) => {
+        await rq({
+          method: 'DELETE',
+          url: `/content-releases/${release.id}`,
+        });
+      })
+    );
+  };
+
+  const createEntry = async (uid, data) => {
+    const { body } = await rq({
+      method: 'POST',
+      url: `/content-manager/collection-types/${uid}`,
+      body: data,
+    });
+
+    return body;
+  };
+
+  beforeAll(async () => {
+    await builder.addContentTypes([productModel]).build();
+    strapi = await createStrapiInstance();
+    rq = await createAuthRequest({ strapi });
+    jest.useFakeTimers();
+    jest.setSystemTime(new Date('2024-01-01T00:00:00.000Z'));
+
+    // Create products
+    await Promise.all([
+      createEntry(productUID, { name: 'Product 1', description: 'Description' }),
+      createEntry(productUID, { name: 'Product 2' }),
+      createEntry(productUID, { name: 'Product 3', description: 'Description' }),
+      createEntry(productUID, { name: 'Product 4', description: 'Description' }),
+      createEntry(productUID, { name: 'Product 5', description: 'Description' }),
+    ]);
+
+    await deleteAllReleases();
```

Comment:
are there any releases before running any tests?

Reply:
Not, but this was really useful while testing (because if you run tests in watch mode they don't clean the database when you made a change)